src/app/api/search/route.ts

```diff
@@ -42,19 +39,6 @@ export async function GET(request: NextRequest) {
       where: {
         type: 'video',
         hidden: false,
-        parent: {
-          courses: {
-            some: {
-              course: {
-                purchasedBy: {
-                  some: {
-                    userId: session?.user?.id,
```

Comment:
@ItsFlash10 you set the cache to a particular user who opens it. This is wrong have to revert